docs/tutorials/gromacs/spack-gromacs.yaml

```diff
@@ -87,17 +103,9 @@ deployment_groups:
               - - $mpi_packages
               - - $%compilers
               - - $^mpis
-      # Un-comment and update mirror_url to install from spack cache
-      # spack_cache_url:
-      # - mirror_name: gcs_cache
-      #   mirror_url: gs://optionally_set_spack_cache_bucket
-
-  - id: controller-setup
-    source: modules/scripts/startup-script
-    settings:
-      runners:
-      - $(spack.install_spack_deps_runner)
-      - $(spack.install_spack_runner)
+      - $(spack.spack_deps_runner)
+      - $(spack.spack_full_install_runner)
+      - $(spack-env.spack_commands_runner)
```

Comment:
This sequence is a little concerning to me. The fact that we need two different spack modules to describe this flow is troubling.